1_5_5/includes/templates/YOUR_TEMPLATE/templates/tpl_modules_attributes.php

```diff
@@ -77,49 +77,41 @@
          } // END h3_option_comment
          ?>
 
-         <div class="wrapperAttribsOptions" id="<?php echo $options_html_id[$I]; ?>">
-              <h4 class="optionName back"><?php echo $options_name[$I]; ?></h4>
-              <div class="back"><?php echo "\n" . $options_menu[$I]; ?></div>
-              <br class="clearBoth" />
-         </div>
+<div class="wrapperAttribsOptions" id="<?php echo $options_html_id[$i]; ?>">
+<h4 class="optionName back"><?php echo $options_name[$i]; ?></h4>
+<div class="back"><?php echo "\n" . $options_menu[$i]; ?></div>
+<br class="clearBoth" />
+</div>
 
 
-         <?php
-         if ($options_comment[$I] != '' and
-                 $options_comment_position[$I] == '1') {
-           ?>
-           <div class="ProductInfoComments"><?php echo $options_comment[$I]; ?></div>
-<?php } // END if Div_options_Comment    
+<?php if ($options_comment[$i] != '' and $options_comment_position[$i] == '1') { ?>
+    <div class="ProductInfoComments"><?php echo $options_comment[$i]; ?></div>
+<?php } ?>
+
+
+<?php
+if ($options_attributes_image[$i] != '') {
+?>
+<?php echo $options_attributes_image[$i]; ?>
+<?php
+}
+?>
+<br class="clearBoth" />
+<?php
+    }
 ?>
-         <?php
-       } // End FOR options_name 
-       ?>
-       <?php
-       // This displays ALL images regardless of attribute stock levels. Comment-out the "echo" if you want to skip images.
-       for ($i = 0; $i < sizeof($options_name); $i++) {
```

Comment:
This was added to support multiple product attribute images which is or related to an add-on in the ZC downloads.  At the time of inclusion it did not seem like keeping the for loop would be an issue, is there something about it that you have seen that is problematic? If so would think that should incorporate an admin control for it, because in the situation of having multiple attributes, this code becomes very useful for changing the product/attribute display image.